packages/fast-development-site-react/src/components/site/action-bar.tsx

```diff
@@ -116,8 +123,11 @@ class ActionBar extends React.Component<IActionBarProps & IManagedClasses<IActio
         );
     }
 
-    private getActionBarMenuButtonClassNames(): string {
-        if (this.props.formView) {
+    private getActionBarMenuButtonClassNames(type: string): string {
+        if (
+            (this.props.formView && type === "configure") ||
```

Comment:
"configure" and "view" look like good candidates for an `enum` - or maybe a few enums